ios/src/TGMapViewController.h

```diff
@@ -12,6 +12,26 @@
 #import <UIKit/UIKit.h>
```

Comment:
OK, after looking further into it. I don't think there is a solution to removing `cc159079` from the rebased branch without actually removing the diff that that commit applies.

I went ahead and made a new branch and applied a diff of the `minZoom` branch.

https://github.com/hallahan/tangram-es/tree/display_zoom

We can a) open a new PR with that, or b) just merge this PR as-is.

Sorry for the weird rebase.